flowzone.yml

```
@@ -54,6 +54,23 @@
       echo "semver=$(npx -q -y -- semver -c -l "${tag}")" >> $GITHUB_OUTPUT
       echo "describe=$(git describe --tags --always --dirty | cat)" >> $GITHUB_OUTPUT
 
+
+  - &generatePythonMetadata
+    name: Generate Python metadata
+    id: python_meta
+    run: |
+      package=$(poetry version --no-ansi | awk '{print $1}')
+      version=$(poetry version --no-ansi | awk '{print $2}')
+      commit_sha=${{ github.event.pull_request.head.sha }}
+      commit_sha=$(echo $commit_sha | tr "a-z" "A-Z")
```

These two lines can be combined, and still be readable I think.
```suggestion
      commit_sha="$(echo ${{ github.event.pull_request.head.sha }} | tr "a-z" "A-Z")"
```

---

done